### PR TITLE
feat: get zoomSelf from options

### PR DESCRIPTION
**Pull request by DanielS-Qlik**
Get `zoomSelf` from options and use as fullscreen callback.
Add `isSmallDevice` to `IEnv`.
Don't render `Resizable` if we are in sense-client. (By checking if `onFullscreen` is present).

### CODE REVIEW

File: src/components/ListboxGrid/ListboxGrid.tsx [hunk 1]
```diff
@@ -62,18 +65,18 @@ export default function ListboxGrid(props: ListboxGridProps) {
...
 
-  const handleOnFullscreen = () => {
-    const { model } = store.getState();
-    if (model) {
-      onFullscreen?.(model.id);
-    }
-  };
+  useEffect(() => {
+    setDisableExpandButton(!!constraints?.active);
+  }, [constraints?.active]);
```
Comment (johanlahti) [inline src/components/ListboxGrid/ListboxGrid.tsx:70]: Can we put `constraints?.active` directly into the returned HTML and get rid of the `useEffect`?

File: src/hooks/use-render.ts [hunk 2]
```diff
@@ -41,11 +37,20 @@ export default function useRender({ flags }: IRenderArgs) {
...
+        onFullscreen: options.zoomSelf,
+        constraints,
+      },
+    );
     return (() => {
       teardown(root);
-    })
-  }, [resourcesArr]);
+    });
+  }, [resourcesArr, constraints, sense]);
```
Comment (johanlahti) [inline src/hooks/use-render.ts:55]: Do we need to listen to `sense` also?

File: src/components/ListboxGrid/ListboxGrid.tsx (hunk 1 above)
Comment (DanielS-Qlik) [reply src/components/ListboxGrid/ListboxGrid.tsx:70]: Yes, nice

File: src/hooks/use-render.ts (hunk 2 above)
Comment (DanielS-Qlik) [reply src/hooks/use-render.ts:55]: I forgot to remove 👍

**Pull request merged**